### DISCUSSION TITLE
Future of SVG support

### DISCUSSION THREAD

**Discussion by ensconced** [antvis/G6#3942] [Q&A] [answered]
It seems like there are lots of issues relating to SVG which aren't getting a lot of attention. What is the best way forward for full support of SVG? I notice G6 is currently using an old version of g-svg/g-canvas. Would it be a good idea to upgrade to the latest versions of g-svg/g-canvas?

**Comment by hustcc** [comment, marked as answer]
We will release 5.0 version in 2024.06, with the latest versions of g-svg/g-canvas.
